firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressLocalization.swift

```diff
@@ -0,0 +1,120 @@
+    let autofillAddressParish: String
+    let autofillAddressPrefecture: String
+    let autofillAddressArea: String
+    let autofillAddressDoSi: String
+    let autofillAddressDepartment: String
+    let autofillAddressEmirate: String
+    let autofillAddressOblast: String
+    let autofillAddressPin: String
+    let autofillAddressEircode: String
+    let autofillAddressCountryOnly: String
+    let autofillCancelButton: String
+    let autofillSaveButton: String
+
+    enum CodingKeys: String, CodingKey {
+        case autofillAddressGivenName = "autofill-address-given-name"
+        case autofillAddressAdditionalName = "autofill-address-additional-name"
+        case autofillAddressFamilyName = "autofill-address-family-name"
+        case autofillAddressName = "autofill-address-name"
+        case autofillAddressOrganization = "autofill-address-organization"
+        case autofillAddressStreet = "autofill-address-street"
+        case autofillAddressState = "autofill-address-state"
+        case autofillAddressProvince = "autofill-address-province"
+        case autofillAddressCity = "autofill-address-city"
+        case autofillAddressCountry = "autofill-address-country"
+        case autofillAddressZip = "autofill-address-zip"
+        case autofillAddressPostalCode = "autofill-address-postal-code"
+        case autofillAddressEmail = "autofill-address-email"
+        case autofillAddressTel = "autofill-address-tel"
+        case autofillEditAddressTitle = "autofill-edit-address-title"
+        case autofillAddressNeighborhood = "autofill-address-neighborhood"
+        case autofillAddressVillageTownship = "autofill-address-village-township"
+        case autofillAddressIsland = "autofill-address-island"
+        case autofillAddressTownland = "autofill-address-townland"
+        case autofillAddressDistrict = "autofill-address-district"
+        case autofillAddressCounty = "autofill-address-county"
+        case autofillAddressPostTown = "autofill-address-post-town"
+        case autofillAddressSuburb = "autofill-address-suburb"
+        case autofillAddressParish = "autofill-address-parish"
+        case autofillAddressPrefecture = "autofill-address-prefecture"
+        case autofillAddressArea = "autofill-address-area"
+        case autofillAddressDoSi = "autofill-address-do-si"
+        case autofillAddressDepartment = "autofill-address-department"
+        case autofillAddressEmirate = "autofill-address-emirate"
+        case autofillAddressOblast = "autofill-address-oblast"
+        case autofillAddressPin = "autofill-address-pin"
+        case autofillAddressEircode = "autofill-address-eircode"
+        case autofillAddressCountryOnly = "autofill-address-country-only"
+        case autofillCancelButton = "autofill-cancel-button"
+        case autofillSaveButton = "autofill-save-button"
+    }
+
+    // Change the correct strings after UX finalize copyright
+    static let editAddressLocalizationIDs = EditAddressLocalization(
```

Comment:
Two small questions
a) Real strings will be in the Strings.swift right?
b) If a) is true can we add a ticket for todo here as well

Reply:
a) Correct
Adding TODO://